Source/ACE.Server/WorldObjects/Managers/EmoteManager.cs

```diff
@@ -1304,6 +1306,13 @@ public void Enqueue(BiotaPropertiesEmote emoteSet, WorldObject targetObject, int
             if (emoteSet == null)
             {
                 Nested--;
+
+                if (OnDeathEmoteInProgress)
```

Comment:
this branch should technically be unneeded

what this should be doing, is simply setting OnDeathEmoteInProgress = HasEmoteCategory(EmoteCategory.Death);

Reply:
actually, this part is completely unsafe, and should be removed